src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/Tables/ServiceRecipient.sql

```diff
@@ -0,0 +1,9 @@
+﻿CREATE TABLE dbo.ServiceRecipient
+(
+    ServiceRecipientId INT IDENTITY(1, 1) NOT NULL,
+    OrderId NVARCHAR(10) NOT NULL,
+    [Name] NVARCHAR(256) NULL, 
+    OdsCode NVARCHAR(8) NULL, 
+    CONSTRAINT PK_ServiceRecipient PRIMARY KEY (ServiceRecipientId),
+    CONSTRAINT FK_ServiceRecipient_Order FOREIGN KEY (OrderId) REFERENCES [Order] (OrderId)

```

Comment:
Normally I would include the column name in the constraint as well. But I don't think this was mentioned in the coding standards meeting, so welcome to ignore. Just thinking could there ever be a case when we have two constraints to the same table but different fields, probably unlikely